L1Trigger/TrackerTFP/interface/State.h

```diff
@@ -29,8 +29,32 @@ namespace trackerTFP {
           const TTBV& maybe,
           int trackId);
     // updated state constructor
-    State(State* state, const std::vector<double>& doubles);
-    // combinatoric and seed building state constructor
+    State(State* state,
+          double x0,
```

Comment:
Why lots of individual variables, rather than vectors?

Reply:
I can't overload two constructors with identical argument list, both would like to a vector<double>, therefore I use individual variables.